tests/conftest.py

```diff
@@ -224,109 +175,76 @@ def check_socket(address, port):
-
-    def is_relevant_fixture_file(f):
-        return any([fnmatch.fnmatch(f, pattern) for pattern in suffixes])
+def mirror_contract_data(dest_file_path: Path, src_file_path: Path, src_network: str, dest_network: str):
+    with open(src_file_path, "r") as file:
+        src_data = json.load(file)
 
-    for f in filter(is_relevant_fixture_file, os.listdir(FIXTURE_FILES_DIR)):
-        os.remove(f"{FIXTURE_FILES_DIR}{f}")
-        logging.info(f"Removed file {FIXTURE_FILES_DIR}{f}")
+    with open(dest_file_path, "r") as file:
+        dest_data = json.load(file)
 
-    # we copy and rename the files according to the expected file name format and destination folder
-    node_nr = 1
-    id_files = sorted(os.listdir(PREGENERATED_IDENTITIES_DIR))
-
-    def is_relevant_id_file(f):
-        return fnmatch.fnmatch(f, "*.id")
-
-    for f in filter(is_relevant_id_file, id_files):
-        shutil.copyfile(
-            f"{PREGENERATED_IDENTITIES_DIR}/{f}", f"{FIXTURE_FILES_DIR}{FIXTURE_FILES_PREFIX}-node_{node_nr}.id"
-        )
-        node_nr += 1
-
-
-def create_local_safes(nodes_args: dict, private_key):
-    custom_env = {
-        "ETHERSCAN_API_KEY": "anykey",
-        "IDENTITY_PASSWORD": PASSWORD,
-        "DEPLOYER_PRIVATE_KEY": private_key,
-        "PRIVATE_KEY": private_key,
-        "PATH": os.environ["PATH"],
+    network_data = src_data["networks"][src_network]
+    partial_network_data = {
+        "environment_type": network_data["environment_type"],
+        "indexer_start_block_number": 1,
+        "addresses": network_data["addresses"],
     }
-    for node_id, node_args in nodes_args.items():
-        id_file = f"{node_args['dir']}.id"
-        res = subprocess.run(
-            [
-                "hopli",
-                "create-safe-module",
-                "--network",
-                ANVIL_NETWORK1,
-                "--identity-from-path",
-                id_file,
-                "--contracts-root",
-                "./ethereum/contracts",
-                "--hopr-amount",
-                "20000.0",
-            ],
-            env=os.environ | custom_env,
-            check=True,
-            capture_output=True,
-            text=True,
+    new_network_data = dest_data["networks"][dest_network] | partial_network_data
+    dest_data["networks"][dest_network] = new_network_data
+
+    with open(dest_file_path, "w") as file:
+        json.dump(dest_data, file, sort_keys=True)
+
+
+def copy_identities():
+    # Remove old identities
+    for f in FIXTURES_DIR.glob(f"{FIXTURES_PREFIX}-*.id"):
+        os.remove(f)
+    logging.info(f"Removed '*.id' files in {FIXTURES_DIR}")
+
+    # Remove old logs
+    for f in FIXTURES_DIR.glob(f"{FIXTURES_PREFIX}-*.log"):
+        os.remove(f)
+    logging.info(f"Removed '*.log' files in {FIXTURES_DIR}")
+
+    # Remove old db
+    for f in FIXTURES_DIR.glob(f"{FIXTURES_PREFIX}-*"):
+        if not f.is_dir():
+            continue
+        shutil.rmtree(f, ignore_errors=True)
```

Comment:
```suggestion
    for f in FIXTURES_DIR.glob(f"{FIXTURES_PREFIX}-*") if f.is_dir():
        shutil.rmtree(f, ignore_errors=True)
```

Reply:
Only allowed in list comprehension, not in standard for loops.